desc/__main__.py

```diff
@@ -44,12 +44,6 @@ def main(cl_args=sys.argv[1:]):
     )
 
     if ir.args.plot > 1:
-        print("Plotting initial guess")
```

Comment:
Any major reason for getting rid of this?

Reply:
I sort of liked this attribute, and my vote is to have it again, unless there was a good reason for getting rid of it. I think knowing the initial guess is good (though I guess if you are changing a single equilibrium a lot the initial guess gets less and less important)

Reply:
To answer @f0uriest's question: we had to get rid of plotting the initial guess because `Equilibrium` no longer has the `initial` or `x` attributes. 
To answer @dpanici's question: we got rid of those attributes because the state vector `x` now depends on the `ObjectiveFunction` that is used to optimize it. The `Equilibrium` class no longer "owns" the optimization process, which is the whole point of this pull request. The functionality can still be recovered by the user by recording a copy of the initial guess before solving it.